app/uk/gov/hmrc/gform/sharedmodel/formtemplate/SummarySection.scala

```diff
@@ -49,5 +50,6 @@ case class SummarySection(
 }
 
 object SummarySection extends JsonUtils {
-  implicit val format: OFormat[SummarySection] = Json.format[SummarySection]
+  val jsWithDefaults = Json.using[Json.WithDefaultValues]
+  implicit val format: OFormat[SummarySection] = jsWithDefaults.format[SummarySection]
```

Comment:
We should be using derived format on frontend.  Why is `Json.format[SummarySection]` not good enough?

Reply:
it is derived, but with support of default value of `displayWidth`

Reply:
But we don't need default value support, since this value will always be populated to us from the backend.

Reply:
good point, I reverted to the original format